### DISCUSSION TITLE
Wallet problem

### DISCUSSION THREAD

**Discussion by inkone2433** [Chia-Network/chia-blockchain#12698] [General]
downloaded the wallet, sent my chia tokens to it, but the wallet is constantly synchronizing, which does not end, deleted completely, downloaded again, nothing helps, I tried all possible methods from the team, I don’t have the patience to wait, where are my tokens, how to solve it ?This not normal!

**Comment by johnathon-b** [comment]
It takes awhile for the wallets to sync. Are you running a full node or a lite wallet?

You could try using Goby browser or another wallet if you cannot wait for the wallet to sync. However I recommend you let it sync.
